site/mkdocs.yml

```diff
@@ -29,7 +29,10 @@ nav:
 - Reference:
   - reference/crds.md
   - Connectors API: reference/connectors.md
```

Comment:
I think that we can remove it now (and change the title of the new docs to just `Connectors API`).

Reply:
fixed. Please check.